### PR TITLE
Fixes the Chef not having CQC outside the kitchen on Tramstation

### PR DESCRIPTION
**Pull request by E231kei500bandai**
## About The Pull Request

The additional areas were broken in #68084 because the area around the bar was changed from /area/station/service/kitchen/diner to /area/station/commons/lounge. However, the cqc addtional areas was never updated to reflect this change in the area.

I found this while looking back at #73878 (Were they just doing another coder meme I don't know about or something?) 

The visible areas are where the Chef gets CQC now

image

It might be a bit big, but compared to how big the area is on other maps I think it's fine.   

Additionally, removes additional cqc areas being defined as /area/station/service/kitchen/diner for icebox for a minor code improvement, it was pointless since the diner is a subtype of kitchen

## Why It's Good For The Game

Chef has the CQC in the area they should

They're supposed to be able to protect the bot portal with it. 

## Changelog
:cl:
fix: Fixed the Chef not having CQC outside the kitchen on Tramstation
/:cl:

### CODE REVIEW

**Comment by BlueMemesauce** [comment]
Also North Star seems to not have any additional cqc areas at all, but I guess that's intentional?

**Comment by TheVekter** [comment]
Can we get a screenshot of where this adds CQC for the cook?

**Comment by san7890** [comment]
*mumble* map configs might drift from the actual reality of the map

@MMMiracles thoughts?

**Comment by TheVekter** [comment]
I've always been under the impression that cook only gets CQC in the physical kitchen itself as well as any backrooms, ie "areas only he has access to". If we're talking about the WHOLE LOUNGE, that's entirely too much room.

**Comment by BlueMemesauce** [comment]
Yes but then we added additional access to the area in front of the kitchen so the chef can defend the restaurant bots.

File: _maps/icebox.json [hunk 1]
```diff
@@ -46,9 +46,6 @@
 	"minetype": "none",
 	"blacklist_file": "iceruinblacklist.txt",
 	"job_changes": {
-		"Cook": {
-			"additional_cqc_areas": ["/area/station/service/kitchen/diner"]
```
Comment (JohnFulpWillard) [inline _maps/icebox.json:50]: Chefs have access to all kitchen areas, right?

Comment (JohnFulpWillard) [reply _maps/icebox.json:50]: Oh Melbert commented about this already

**Pull request merged**